Sources/Sentry/SentryCoreDataTracker.m

```diff
@@ -109,7 +109,7 @@ - (BOOL)managedObjectContext:(NSManagedObjectContext *)context
         [fetchSpan finishWithStatus:result ? kSentrySpanStatusOk : kSentrySpanStatusInternalError];
 
         SENTRY_LOG_DEBUG(@"SentryCoreDataTracker automatically finished span with status: %@",
-            *error == nil ? @"ok" : @"error");
+            result ? @"ok" : @"error");
```

Comment:
If `result` is true, then the call succeeded, and we should log `ok`. We have no guarantee about the state of `error` in this case.

If `result` is false, then there was an error. *If* the user passed in an `error` pointer, it will now point to the error. But if the user passed in nil, then `error` will still be nil, and dereferencing it would cause a crash. In either case, we don't need the actual details of the error here at all; we only need to know _whether_ an error happened, and that can be determined entirely by looking at the result.